data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -152,11 +150,31 @@ $IPTABLES -t raw -A DHCP_IRONIC -j DROP
 {{end}}
 
 # Wait for images to be downloaded/ready
-podman wait -i 1000ms ipa-downloader
+#podman wait -i 1000ms ipa-downloader
 podman wait -i 1000ms coreos-downloader
-while ! curl --fail http://localhost/images/rhcos-ootpa-latest.qcow2.md5sum ; do sleep 1; done
-while ! curl --fail --head http://localhost/images/ironic-python-agent.initramfs ; do sleep 1; done
-while ! curl --fail --head http://localhost/images/ironic-python-agent.kernel ; do sleep 1; done
+
+# Write master network config to /tmp/nmstate for image_customization_controller to serve our merged images
+mkdir -p /tmp/nmstate
+{{range .PlatformData.BareMetal.Hosts}}
+    cat >/tmp/nmstate/{{.Name}}.yaml <<EOF
+{{.NetworkConfig}}
+EOF
+{{end}}
+
+CUSTOMIZATION_IMAGE="virthost.ostest.test.metalkube.org:5000/localimages/image-customization-controller"
+
+# Embed agent ignition into the rhcos live iso
+sudo podman run -d --net host --privileged --name image-customization \
+    --env DEPLOY_ISO="/shared/html/images/ironic-python-agent.iso" \
```

Comment:
Do we need net=host and privileged here?

Reply:
hah! probably not.. I'm so used to it now :).